backend/app/services/translations/translate_content.rb

```diff
@@ -0,0 +1,26 @@
+module Translations
+  class TranslateContent
+    def initialize(translatable_content, source_language_code)
+      @translatable_content = translatable_content
+      @source_language_code = source_language_code
+    end
+
+    def client
+      @client = Google::Cloud::Translate.translation_service
+    end
+
+    def call(target_language_code)
+      response = client.translate_text contents: @translatable_content,
+        source_language_code: @source_language_code,
+        target_language_code: target_language_code,
+        parent: "projects/#{ENV["GCP_PROJECT_ID"]}/locations/global",
```

Comment:
looks like this depends on `projects` entity, even though it looks like same base class for content translations.